### DISCUSSION TITLE
Plugin documentation

### DISCUSSION THREAD

**Discussion by MacroMan** [adminerevo/adminerevo#64] [Ideas]
Hi, new in this community, but a user of Adminer for many years.

I notice that documentation for plugins on the AdminerEvo site is lacking, and this is something I am willing to help with.

I suggest a port of the docs from https://www.adminer.org/en/plugins/ as a starting place to make further improvements.

Opening up discussion and keen to hear your thoughts 

David

**Comment by LionelLaffineur** [comment]
Hi @MacroMan, this sounds like a good idea !  Listing the plugins and describing their use sounds a good plan !
If you would like to post something, I'd be happy to add it to the repository or feel free to do a pull request if you prefer.
Regards,
L.

**Reply by MacroMan** [reply]
I've opened a PR for the initial migration: https://github.com/adminerevo/adminerevo/pull/65

There will need to be improvements made, especially around usage instructions, since the original instructions are a bit vague, but I think this is a good start.
